src/removeTopDependency.ts

```diff
@@ -22,13 +23,15 @@ export default async function removeTopDependency (
   ])
 
   const uninstalledPkg = results[0]
-  rootLogger.info({
-    removed: {
-      name: dependency.name,
-      version: uninstalledPkg && uninstalledPkg.version,
-      dependencyType: dependency.dev && 'dev' || dependency.optional && 'optional' || 'prod'
-    },
-  })
+  if (!opts.skipLogs) {
```

Comment:
it might be useful to have logs anyway but the reporter should be able to differentiate the hoisted dependencies from the regular once, so maybe lets' keep the root logs but with an additional field `hoisted: Boolean`.